src/main/java/com/shipmentEvents/demo/createCookie.java

```diff
@@ -0,0 +1,8 @@
+import javax.servlet.http.Cookie;
+
+public static void createCookie() {
+  Cookie cookie = new Cookie("name", "value");
+    cookie.setMaxAge(60 * 60 * 24 * 365);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;;&#34;, @ expected